apps/communications/dialer/test/unit/suggestion_bar_test.js

```diff
@@ -405,20 +401,14 @@ suite('suggestion Bar', function() {
       });
 
       test('should load the overlay', function() {
-        sinon.assert.calledWith(LazyLoader.load, domOverlay);
+        sinon.assert.called(LazyLoader.load);
       });
 
       test('overlay is displayed', function() {
-        assert.equal(subject.overlay.getAttribute('aria-hidden'), 'false');
+        assert.equal(subject.overlay.hasAttribute('hidden'), false);
         assert.isTrue(subject.overlay.classList.contains('display'));
       });
 
-      test('should have a cancel button as the last button', function() {
```

Comment:
This test is no longer needed as the cancel button is button is behind the shadowRoot, and gaia_menu has its own tests.